docs/book/src/cronjob-tutorial/testdata/project/api/v1/cronjob_webhook.go

```diff
@@ -75,6 +75,8 @@ A webhook will automatically be served that calls this defaulting.
 The `Default` method is expected to mutate the receiver, setting the defaults.
 */
 
+// +kubebuilder:webhook:path=/mutate-batch-tutorial-kubebuilder-io-v1-cronjob,mutating=true,failurePolicy=fail,sideEffects=None,groups=batch.tutorial.kubebuilder.io,resources=cronjobs,verbs=create;update,versions=v1,name=mcronjob-v1.kb.io,admissionReviewVersions=v1
+
```

Comment:
A little newline bug here?  Shouldn't the +kubebuilder directives be togeher?

Reply:
One marker is for the Mutating Webhook, and the other is for the Validating Webhook. Each webhook is scaffolded with its respective code—one for mutation and the other for validation. So, it's not a bug.